sys/net/gnrc/network_layer/sixlowpan/frag/rb/gnrc_sixlowpan_frag_rb.c

```diff
@@ -566,14 +566,14 @@ static void _gc_pkt(gnrc_sixlowpan_frag_rb_t *rbuf)
 
 void gnrc_sixlowpan_frag_rb_gc(void)
 {
-    uint32_t now_usec = xtimer_now_usec();
+    uint32_t now_msec = ztimer_now(ZTIMER_MSEC);
     unsigned int i;
 
     for (i = 0; i < CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_SIZE; i++) {
         /* since pkt occupies pktbuf, aggressivly collect garbage */
         if (!gnrc_sixlowpan_frag_rb_entry_empty(&rbuf[i]) &&
-              ((now_usec - rbuf[i].super.arrival) >
-               CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US)) {
+              ((now_msec - rbuf[i].super.arrival) >=
+               CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US / US_PER_MS)) {
```

Comment:
This macro is only used within the reassembly buffer, so why not redefine it to `TIMEOUT_MS`. If you want to avoid API changes, we could define a deprecated `TIMEOUT_US = TIMEOUT_MS * US_PER_MS` as well.

Reply:
I actually had to deprecate 3 config vars.

[94349f4](https://github.com/RIOT-OS/RIOT/pull/19067/commits/94349f4e2dead4f0a97361b21bba9afdaf2c0aba)

To get things clear I prefixed the time unit to some affected variables.

If you're happy with the changes, I'd squash if you don't mind.